PyInstaller/compat.py

```diff
@@ -169,6 +169,11 @@
 base_prefix = os.path.abspath(base_prefix)
 is_venv = is_virtualenv = base_prefix != os.path.abspath(sys.prefix)
 
+# Conda environments sometimes have different paths or apply patches to
+# packages that can affect how a hook or package should access resources.
+# Method for determining conda taken from:
+# https://stackoverflow.com/a/21318941/433202
+is_conda = 'conda' in sys.version or 'Continuum' in sys.version
```

Comment:
This is a good idea! Please but it into a commit of it's own.

Reply:
Of course. I probably would have done that if I wasn't in such a rush.